llvm/test/Analysis/CtxProfAnalysis/load-unapplicable.ll

```diff
@@ -0,0 +1,60 @@
+; REQUIRES: x86_64-linux
+;
+; RUN: rm -rf %t
+; RUN: split-file %s %t
+; RUN: llvm-ctxprof-util fromJSON --input=%t/profile.json --output=%t/profile.ctxprofdata
+; RUN: opt -passes='require<ctx-prof-analysis>,print<ctx-prof-analysis>' -ctx-profile-printer-level=everything \
+; RUN:   %t/example.ll -S 2>&1 | FileCheck %s
+
+; CHECK: No contextual profile was provided
```

Comment:
Should the error be a bit more explicit - e.g. "No contextual profile containing functions from this module was provided" or something like that?